modules/generic_worker/templates/run-generic-worker.sh.erb

```diff
@@ -14,6 +14,39 @@ rm -f tasks-resolved-count.txt
 # wrapper script...
 /usr/local/bin/generic-worker "$@" 2>&1 | logger -t generic-worker -s
 
+exitstatus=( ${PIPESTATUS[0]} )
+
+echo $exitstatus
```

Comment:
I don't think we need this `echo`, the worker already logs its exit status. If you choose to keep it, please log that it is the generic-worker exit code (e.g. `echo "Generic worker exit code was ${exitstatus}"`).

Reply:
petemoore: You are right. I used this echo for debug, I'll remove it

Reply:
720 hours and 5 seconds means 30 days and 5 second. I think is enough time to someone have a look and fix the worker